app/src/main/java/com/fstyle/androidtrainning/screen/main/fragments/online/OnlineFragment.java

```diff
@@ -0,0 +1,47 @@
+package com.fstyle.androidtrainning.screen.main.fragments.online;
+
+import android.os.Bundle;
+import android.support.design.widget.TabLayout;
+import android.support.v4.app.Fragment;
+import android.support.v4.view.ViewPager;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+
+import com.fstyle.androidtrainning.R;
+import com.fstyle.androidtrainning.screen.main.fragments.mysong.MySongViewPagerAdapter;
```

Comment:
ERROR: Unused import: com.fstyle.androidtrainning.screen.main.fragments.mysong.MySongViewPagerAdapter. Use Ctrl+Shift+O (Eclipse) or Ctrl+Alt+O (Intellij) to sort imports